src/ONYXKEYS.js

```diff
@@ -161,6 +161,9 @@ export default {
     // Is report data loading?
     IS_LOADING_REPORT_DATA: 'isLoadingReportData',
 
+    // Is server side report search loading?
+    IS_LOADING_SERVER_SEARCH_REPORT_DATA: 'isLoadingServerSearchReportData',
```

Comment:
NAB, it's possible we don't need to show an indicator for this yet.

Reply:
Removed, we can add it later if needed